### ISSUE TITLE
Tab scrolling doesn't work when mouse between tabs

### ISSUE THREAD

**Issue by jupjohn** [Chatterino/chatterino2#2970]
**Describe your issue**
If the user scrolls while their mouse is in between tabs (1px width) the current tab won't change. This also happens on the deadzone to the right of horizontal tabs.

**OS and Chatterino Version**
Arch Linux (5.12.14-zen1), bspwm + picom
Chatterino 2.3.3 (commit cc9d44a0)

**Comment by jammehcow** [comment]
Fix could also tie in to #2916

**Comment by zneix** [comment]
I think this is intended - mouse cursor isn't over tabs. Events responsible for tab scrolling are registered in `NotebookTab::wheelEvent` while that little space between tabs isn't a part of `NotebookTab`.
Though, as you mentioned it could be solved with a fix for #2916 so it makes sense to keep this open.

**Comment by jammehcow** [comment]
I'm not too concerned about the empty space scrolling - it's that 1px between tabs that breaks the feel. Still not sure how you'd fix that without doing that background too.

**Comment by zneix** [comment]
Yes I think we'd need to enable scrolling over background or something similar.
